build/heroes.json

```diff
@@ -11,7 +11,21 @@
       "Nuker"
     ],
     "img": "/apps/dota2/images/heroes/antimage_full.png?",
-    "icon": "/apps/dota2/images/heroes/antimage_icon.png"
+    "icon": "/apps/dota2/images/heroes/antimage_icon.png",
+    "url": "Anti-Mage",
+    "base_armor": "-1",
```

Comment:
it's what is in the d2vpkr, I imagine it doesn't include extra armor from agility